src/stories/common.ts

```diff
@@ -14,6 +14,7 @@ export const ExcludeReactFormHookProps = {
 };
 
 export const CommonFieldStoryProps = (fieldType: string) => ({
+	[`${fieldType}-default`]: { table: { disable: true } },
 	fieldType: {
```

Comment:
can help me add props for `validation` and `showIf`? 
ref: https://github.com/LifeSG/web-frontend-engine/commit/7fe491e94182e91f5c40a95a4d34b5a272481118#diff-aa249136a8c16ecec20779866c9ab5c889567972fc97125392130206a90e6a0a